src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
```

Comment:
The `Lottery` class should not extend `Ball`. Inheritance implies an 'is-a' relationship, but a `Lottery` is not a `Ball`. It would be more appropriate for `Lottery` to have a method that returns a `Ball` object without extending the `Ball` class.